### ISSUE TITLE
Cannot run CliWizardIntegrationTest

### ISSUE THREAD

**Issue by LorenzoBettini** [eclipse/xtext-core#1304]
I cannot run `org.eclipse.xtext.xtext.wizard.cli.CliWizardIntegrationTest` as a JUnit test, due to this exception from EMF: `Missing properties: org/eclipse/xtext/util/plugin.properties`.

I can run it as a JUnit Plug-in test, but then I cannot run its main to update the expectations:

```
Exception in thread "main" java.lang.ExceptionInInitializerError
Caused by: java.util.MissingResourceException: Missing properties: org/eclipse/xtext/util/plugin.properties
	at org.eclipse.emf.common.util.DelegatingResourceLocator.getBaseURL(DelegatingResourceLocator.java:200)
	at org.eclipse.xtext.util.XtextVersion.readVersionFromManifest(XtextVersion.java:96)
	at org.eclipse.xtext.util.XtextVersion.getCurrent(XtextVersion.java:42)
	at org.eclipse.xtext.xtext.wizard.WizardConfiguration.<init>(WizardConfiguration.java:50)
	at org.eclipse.xtext.xtext.wizard.cli.CliWizardIntegrationTest$1.<init>(CliWizardIntegrationTest.java:390)
	at org.eclipse.xtext.xtext.wizard.cli.CliWizardIntegrationTest.newProjectConfig(CliWizardIntegrationTest.java:390)
	at org.eclipse.xtext.xtext.wizard.cli.CliWizardIntegrationTest.<clinit>(CliWizardIntegrationTest.java:150)
```

**Comment by cdietrich** [comment]
have never seen this. how do you exactly run the test.

**Comment by LorenzoBettini** [comment]
Simply from Eclipse: right-click -> Run As -> JUnit Test

similarly for running its main as a standard Java application

**Comment by cdietrich** [comment]
@szarnekow can it be this is a side effect of the build.properties thing?

**Comment by cdietrich** [comment]
i can reproduce this now too.
was working until last week

**Comment by LorenzoBettini** [comment]
Indeed I've been using that test lots of time in the past and it was always working...

**Comment by cdietrich** [comment]
@kthoms is it possible the xtend2java conversion changed exception handling here?

**Comment by cdietrich** [comment]
indeed the catch exception in xtextversion was not ported properly

**Comment by cdietrich** [comment]
@kthoms this should be fixed for 2.20

**Comment by kthoms** [comment]
will do!

**Comment by kthoms** [comment]
Fixed for 2.20. The regression uncovered a problem in `XtextVersion#readVersionFromManifest()` which might be solved later. The PR #1309 is not ready to merge yet and a bit risky so close to the release.

**Comment by LorenzoBettini** [comment]
Thanks! It now works again
